go/vt/vtgate/vtgate.go

```diff
@@ -192,6 +193,10 @@ func Init(ctx context.Context, serv srvtopo.Server, cell string, tabletTypesToWa
 			"VtgateApiRowsReturned",
 			"Rows returned through the VTgate API",
 			[]string{"Operation", "Keyspace", "DbType"}),
+		rowsWritten: stats.NewCountersWithMultiLabels(
+			"VtgateApiRowsWritten",
```

Comment:
I suggest we call it `VtgateApiRowsAffected` inline with mysql affected rows.

Reply:
I see that in LegacyInit method it is named as `VtgateApiRowsAffected`, so we should keep one. I vote for `VtgateApiRowsAffected`

Reply:
I had started with `VtgateApiRowsAffected`, and because of the bug/workaround, I tried to rename it. Obviously I missed one place.